wolox-books/src/app/screens/templates/authentication/authentication.component.scss

```diff
@@ -8,7 +8,7 @@
   border-top: 8px solid $wolox-blue;
   display: flex;
   flex-direction: column;
-  margin: 0 auto;
+  margin: 100px auto 0 auto;
```

Comment:
You can use the shorthand `margin: 100px auto 0;` 😄

Reply:
FIxed